src/gsy_e/models/strategy/infinite_bus.py

```diff
@@ -104,7 +113,9 @@ def _read_or_rotate_profiles(self, reconfigure=False):
                         profile_uuid=self.energy_rate_profile_uuid)))
 
         self._set_global_market_maker_rate()
+        self._set_global_feed_in_tariff_rate()
 
+    # pylint: disable=fixme
```

Comment:
I think that the TODO comments in this file have to either be deleted, or better clarified in order to be better understood.

Reply:
I would suggest to make a refactoring ticket to monitor its relation to memory usage as simply a TODO proved to be overlooked here

Reply:
It got overlooked/ignored because there was no clear action that had to be done here. If there is suspicion that there is a memory leak here, then this needs to be identified. If not, then the comment needs to be deleted and if / when we have a memory leak we will look at it in detail. 
Moreover, from a brief look at the code, this dict is only populated on construction. Therefore even if this `del` fails to deallocate the object due to reference counting, there is no chance that this dict will grow in time, therefore taking into account that in the vast majority of the simulations there is at most one infinite bus, this will never leak.

Reply:
Okay, to conclude, still its seems better to delete an unused profile here, even in this case where almost un-noticeable memory optimisation is being achieved, considering the fact that there is only one `InfiniteBus` in a simulation

Reply:
The del should stay as it is. What I was talking about above was that there cannot be a memory leak, because this variable is only assigned once and never reassigned. Therefore even if the delete does not work, the size of this variable will never grow because there are no more assignments to it. Therefore a memory leak is not possible in this case. 

Therefore, the TODO comments are not needed here. Please delete them and remove the `# pylint: disable=fixme `

Reply:
https://github.com/gridsingularity/gsy-e/pull/1338/commits/46fb16eca6742c005f891c418f990d94fcf665bf